example/android/app/build.gradle

```diff
@@ -100,6 +100,6 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:23.4.0"
-    compile "com.facebook.react:react-native:0.29.0"  // From node_modules
+    compile "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
Also, even if you specify an explicit version like `0.29.0`, gradle will automatically switch to a newer version if any of your other dependencies depends on a newer version of `react-native`